psi4/driver/procrouting/dft/hyb_functionals.py

```diff
@@ -603,6 +603,54 @@ def get_pw6b95_tweaks():
     "doi": "10.1021/acs.jctc.8b00072",
 })
 
+funcs.append({
+    "name": "R2SCAN0",
+    "x_functionals": {
+        "MGGA_X_R2SCAN": {"alpha": 0.75}
+    },
+    "c_functionals": {
+        "MGGA_C_R2SCAN": {}
+    },
+    "x_hf": {
+        "alpha": 0.25
+    },
+    "description": '    r2SCAN0 Hybrid Meta-GGA XC Functional\n',
+    "citation": '    M. Bursch, H. Neugebauer, S. Ehlert, S. Grimme  J. Chem. Phys. 156, 134105, 2022   \n',
+    "doi": "10.1063/5.0086040",
```

Comment:
Just use `XC_HYB_MGGA_XC_R2SCAN0`

Reply:
Right now we're supporting libxc v.5.1.2 ... v6.x . Would that range have to be curtailed to use the all-in-one fctl definitions?

iirc, we ended up supporting two major versions because v6 came out right before a psi4 release, and we didn't want to demand a sudden change. That is, there's no harm in using only v6 now that c-f can build new Windows versions for us.

Reply:
Yeah the r2scan hybrids were added in 6.1.0.

Reply:
ok, let's plan on dropping v5 and requiring minimum v6.1 then.